app/views/ProfileView/index.tsx

```diff
@@ -471,10 +495,46 @@ class ProfileView extends React.Component<IProfileViewProps, IProfileViewState>
 							value={email || undefined}
 							onChangeText={value => this.setState({ email: value })}
 							onSubmitEditing={() => {
-								this.newPassword?.focus();
+								this.nickname?.focus();
 							}}
 							testID='profile-view-email'
 						/>
+						{compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '3.5.0') ? (
+							<FormTextInput
+								inputRef={e => {
+									if (e) {
```

Comment:
This `if` is unnecessary, you probably copied it from the code above.
But it's always good to check what you're copying, why something exists.

Reply:
You can say that the type says to put this, so in these cases the correct thing is to correct the type.